### PR TITLE
Remove snapshot from release/0.17

### PR DESCRIPTION
**Pull request by vsethi09**
Remove snapshot from release/0.17

### CODE REVIEW

File: cdap-authorization-dataset-extn/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>io.cdap.cdap</groupId>
     <artifactId>cdap-security-extensions</artifactId>
-    <version>0.6.0-SNAPSHOT</version>
+    <version>0.17.0</version>
```
Comment (itsankit-google) [inline cdap-authorization-dataset-extn/pom.xml:25]: `0.6.0`?

File: pom.xml [hunk 2]
```diff
@@ -16,12 +16,12 @@
   -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POMcdap-security-extensions/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (itsankit-google) [inline pom.xml]: why this change?

File: cdap-authorization-dataset-extn/pom.xml (hunk 1 above)
Comment (vsethi09) [reply cdap-authorization-dataset-extn/pom.xml:25]: This wasn't updated for a long time. Updating to 0.17.0 now.

File: pom.xml (hunk 2 above)
Comment (vsethi09) [reply pom.xml]: Not intended. Reverted.

**Review by itsankit-google — APPROVED** [review]
